package.json

```diff
@@ -3,15 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@hookform/resolvers": "^0.1.0",
     "bulma": "^0.9.0",
     "classnames": "^2.2.6",
     "date-fns": "^2.15.0",
     "firebase": "^7.18",
+    "mutationobserver-shim": "^0.3.7",
```

Comment:
Why we use this package?

Reply:
There was an issue when testing and the react-hook-form recommend installing it, you can check it [here](https://react-hook-form.com/faqs#TestingfailedduetoMutationObserver).